keyboards/karn/readme.md

```diff
@@ -0,0 +1,99 @@
+# Karn 2
+
+Split, ortholinear, non-staggered, diodeless, 38 key keyboard, with a unique set of 1.5u thumb arc keys and 1.5u pinky reach keys. 
+
+* Keyboard Maintainer: [Robert Mills](https://github.com/robcmills)
+* Hardware Supported: Blackpill STM32F401
+
+Based on the excellent [cantor](https://github.com/diepala/cantor), and inspired by the popular [corne](https://github.com/foostan/crkbd), [ferris](https://github.com/pierrechevalier83/ferris) and [sweep](https://github.com/davidphilipbarr/Sweep) keyboards.
+
+[https://github.com/robcmills/karn-keyboard](https://github.com/robcmills/karn-keyboard)
+
+![karn-2-left](https://github.com/robcmills/karn-keyboard/blob/main/images/karn-2-left.jpg)
+
+![karn-2-full](https://github.com/robcmills/karn-keyboard/blob/main/images/karn-2-full.jpg)
+
+![karn-2-pcb](https://github.com/robcmills/karn-keyboard/blob/main/images/karn-2-pcb.png)
```

Comment:
These images should be rehosted on Imgur.

Reply:
[karn 90dcbd1ff6] Host images on imgur